component/otelcol/config_attraction.go

```diff
@@ -0,0 +1,103 @@
+package otelcol
+
+type AttrActionKeyValueSlice []AttrActionKeyValue
```

Comment:
This pattern is something I made up myself, since it makes sense to use actions as a slice and to have the convert function in config_atraction.go. If you think this pattern doesn't make senes though, let me know